File: translations/codenet/mistral_commenetd/C2P/starcoder/Code/s992866158.py
# The main function is the entry point of the program
def main():
	# Declare variables n, k, and cnt of type int
	n = 0
	k = 0
	cnt = 0

	# Read two integers n and k from the standard input using input()
	n, k = input().split()
	# Convert n and k to integers
	n = int(n)
	k = int(k)
	# Initialize variable cnt to zero
	cnt = 0
	# Loop through n times using a while loop
	while n > 0:
		# Declare variable h of type int
		h = 0

		# Read an integer h from the standard input using input()
		h = int(input())
		# If h is greater than or equal to k, increment cnt by 1
		if h >= k:
			cnt += 1
		# Decrement n by 1
		n -= 1
	# Print the value of cnt to the standard output using print()
	print(cnt)

# Call the main function
if __name__ == "__main__":
	main()

File: data/codenet/mistral commented/C/Code/s992866158.c
#include <stdio.h>

// The main function is the entry point of the program
int main() {
	// Declare variables n, k, and cnt of type int
	int n, k, cnt;

	// Read two integers n and k from the standard input using scanf()
	scanf("%d%d", &n, &k);
	// Initialize variable cnt to zero
	cnt = 0;
	// Loop through n times using a while loop
	while (n--) {
		// Declare variable h of type int
		int h;

		// Read an integer h from the standard input using scanf()
		scanf("%d", &h);
		// If h is greater than or equal to k, increment cnt by 1
		if (h >= k)
			cnt++;
	}
	// Print the value of cnt to the standard output using printf()
	printf("%d\n", cnt);
	// Return 0 to indicate successful execution of the program
	return 0;
}
